### DISCUSSION TITLE
Redbean + Platypus

### DISCUSSION THREAD

**Discussion by ad-si** [jart/cosmopolitan#954] [Ideas]
Wrapping Redbean with [Platypus](https://github.com/sveinbjornt/Platypus) is such a match made in heaven that I feel like it should be mentioned in the documentation.

It can be used like this:

<img width="500" alt="image" src="https://github.com/jart/cosmopolitan/assets/36796532/3dfef563-94d5-4b32-ac5a-6008107adf89">

To build an app:

<img width="300" alt="Screenshot 2023-11-15 at 09 51 10" src="https://github.com/jart/cosmopolitan/assets/36796532/afc82dd8-5710-4dc6-bd3f-8e0a12aee1c7">

And when this app is opened, it automatically opens the browser and you get an app icon in the dock. The app can then be exited like any other macOS app.

<img width="400" alt="Screenshot 2023-11-15 at 09 55 21" src="https://github.com/jart/cosmopolitan/assets/36796532/ecc8c903-5174-47d5-bd36-7f3617d2a851">

**Comment by pkulchenko** [comment]
This is interesting; thank you for sharing. For those interested to do something similar outside of macOS (although, it should work on macOS as well), adding `.args` file makes redbean to apply configuration options specified in that file when it's launched (see https://redbean.dev/#specials), so adding `-w/` to that file will launch a browser with URL pointing at this redbean instance.
